02c_layout.ipynb

```diff
@@ -0,0 +1,738 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "775f9b1e-8921-4517-8feb-5fcb5e69d57a",
+   "metadata": {},
+   "source": [
+    "# Reviewing Jupyter Widgets: Layout of our Data Dashboard\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ea8e075",
+   "metadata": {},
+   "source": [
+    "### Goal of this Notebook\n",
+    "\n",
+    "In this notebook we will be developing a `main.py` module which will actually implement the full data dashboard. To achieve this, we will take our previously developed widgets and use Jupyter widget's layout mechanisms to create a more polished and user-friendly interface.  \n",
+    "\n",
+    "### Steps You Will Take In This Notebook\n",
+    "In this notebook you will:\n",
+    "1. Import our previously developed widgets\n",
+    "2. Experiment with style and layout of widgets\n",
+    "3. Create a layout for our data dashboard\n",
+    "4. Create a function to produce our data dashboard\n",
+    "5. Export the `main.py` module"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5244bb33",
+   "metadata": {},
+   "source": [
+    "Designate that we are exporting selected code to the `main` module."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5d62ce64-3f56-4ac4-8d8b-1eeb94f22c1e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| default_exp main"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91f96da3-1429-4c17-a340-2a6c9fedb1b1",
+   "metadata": {},
+   "source": [
+    "## Importing our Previously Developed Widgets\n",
+    "\n",
+    "Let's start by importing the widgets we created last time for controlling parameters and for displaying the `pandas` dataframe and the plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "88e92874-4371-4342-9658-41fdaa1ecfe6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| export\n",
+    "# %answer key/dashboard/main.py 5\n",
```

Comment:
I would drop this as an exercise...but if you want to keep then I would remove the answer from the cell 😀